tests/core/fs/test_fs_sync.py

```diff
@@ -722,9 +737,13 @@ async def test_sync_data_before_workspace(running_backend, alice_user_fs):
 
 
 @pytest.mark.trio
-async def test_merge_resulting_in_no_need_for_sync(running_backend, user_fs_factory, alice, alice2):
+async def test_merge_resulting_in_no_need_for_sync(
+    running_backend, user_fs_factory, alice: LocalDevice, alice2: LocalDevice
+):
     async with user_fs_factory(alice) as alice_user_fs:
```

Comment:
you don't want to type `user_fs_factory` instead ?

Reply:
That's already done, I don't have pushed the change for it